packages/frontend/src/components/common/Table/Table.styles.ts

```diff
@@ -152,7 +158,9 @@ export const ThActionsContainer = styled.div`
     }
 `;
 
-export const TableHeaderLabelContainer = styled.div``;
+export const TableHeaderLabelContainer = styled.div`
+    white-space: nowrap;
```

Comment:
@PriPatel fyi: You mentioned something about having the headers "shrink" if the table was close to 100% width.  With this change the table header will always be 1 line, even when using the full width table ( table viz and underlying data ).